manifests/master/generate_types.pp

```diff
@@ -94,8 +94,7 @@
     exec { 'simp_generate_types':
       command     => "${_generate_types_path} -d ${delay} -s -p all",
       refreshonly => true,
-      subscribe   => File[$_generate_types_path],
```

Comment:
You probably do want this to run when that file is updated.

Reply:
Sure; we'll try it that way, but if it results in `simp_generate_types` running before `Service['puppetserver']` is started, then it will fail again.